styles/components/_modalWindow.scss

```diff
@@ -1,3 +1,9 @@
+//TO REMOVE!!!!
+html {
+  overflow: hidden;
+}
+
```

Comment:
???

Reply:
Needed to hide the unordered game components under the modal. Removed.